### ISSUE TITLE
False positive in Rails/FilePath when string contains an interpolation followed by a period

### ISSUE THREAD

**Issue by Fustrate** [bbatsov/rubocop#5574]
@koic 

It looks like [this commit](https://github.com/bbatsov/rubocop/commit/707c9a85c1d99bfee8a10ac404abf6a14d6a88f0#diff-ace2b8b8b0231f6d770d8dad5fa01172) causes any string containing an interpolation followed by a period to get flagged by the Rails/FilePath cop.

--------

## Expected behavior

Should pass.

## Actual behavior

```
Inspecting 1 file
C

Offenses:

test.rb:1:6: C: Rails/FilePath: Please use Rails.root.join('path', 'to') instead.
puts "test #{123}. Hey!"
     ^^^^^^^^^^^^^^^^^^^

1 file inspected, 1 offense detected
```

## Steps to reproduce the problem

`echo 'puts "test #{123}. Hey!"' | bundle exec rubocop -s test.rb --only "Rails/FilePath"`

## RuboCop version

master @ 44bbbcc

```
$ rubocop -V
0.52.1 (using Parser 2.5.0.0, running on ruby 2.5.0 x86_64-darwin17)
```

**Comment by koic** [comment]
@Fustrate Thanks for the report. I reproduced this false positive, so I will investigate.
